K8s/api-deployment.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    io.kompose.service: api
+  name: api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      io.kompose.service: api
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        io.kompose.service: api
+    spec:
+      containers:
+        - env:
+#            - name: MICRO_REGISTRY
+#              value: mdns
+#            - name: MICRO_SERVER_ADDRESS
+#              value: :8089
+            - name:  MICRO_REGISTRY
+              value:  kubernetes
+            - name: MICRO_SERVER_ADDRESS           
+              value: 0.0.0.0:8089
+            - name: MICRO_BROKER_ADDRESS
+              value: 0.0.0.0:9091               
+            - name: OS_AUTH_AUTHSTRATEGY
+              value: keystone
+            - name: OS_AUTH_URL
+              value: http://192.168.20.108/identity
```

Comment:
Will there be any script which will modify this IP to host IP?

Reply:
It will go in via some env variable at install time. Will be replacing this with env variable. This was hardcoded for testing.